data/explorations/root_linear_coefficient_theorem.yaml

```diff
@@ -378,14 +378,14 @@ states:
     content:
       audio_translations: {}
       html: "What about something a little harder? \_Can you tell me what the terms\
-        \ are in the expansion of this:<div><oppia-noninteractive-math raw_latex-with-value=\"\
-        &amp;quot;(x-3)(x+2)(x-5)&amp;quot;\"></oppia-noninteractive-math><br><br>If\
-        \ a, b, c, and d are the coefficients of\_<oppia-noninteractive-math raw_latex-with-value=\"\
-        &amp;quot;x^3&amp;quot;\"></oppia-noninteractive-math>,\_<oppia-noninteractive-math\
-        \ raw_latex-with-value=\"&amp;quot;x^2&amp;quot;\"></oppia-noninteractive-math>,\_\
-        <oppia-noninteractive-math raw_latex-with-value=\"&amp;quot;x&amp;quot;\"\
-        ></oppia-noninteractive-math>\_and\_<oppia-noninteractive-math raw_latex-with-value=\"\
-        &amp;quot;1&amp;quot;\"></oppia-noninteractive-math>, respectively, format\
+        \ are in the expansion of this:<div><oppia-noninteractive-math math_content-with-value=\"\
+        {&amp;quot;raw_latex&amp;quot;:&amp;quot;(x-3)(x+2)(x-5)&amp;quot;,&amp;quot;svg_filename&amp;quot;:&amp;quot;&amp;quot;}\"></oppia-noninteractive-math><br><br>If\
+        \ a, b, c, and d are the coefficients of\_<oppia-noninteractive-math math_content-with-value=\"\
+        {&amp;quot;raw_latex&amp;quot;:&amp;quot;x^3&amp;quot;,&amp;quot;svg_filename&amp;quot;:&amp;quot;&amp;quot;}\"></oppia-noninteractive-math>,\_<oppia-noninteractive-math\
+        \ math_content-with-value=\"{&amp;quot;raw_latex&amp;quot;:&amp;quot;x^2&amp;quot;,&amp;quot;svg_filename&amp;quot;:&amp;quot;&amp;quot;}\"></oppia-noninteractive-math>,\_\
+        <oppia-noninteractive-math math_content-with-value=\"{&amp;quot;raw_latex&amp;quot;:&amp;quot;x&amp;quot;,&amp;quot;svg_filename&amp;quot;:&amp;quot;&amp;quot;}\"\
+        ></oppia-noninteractive-math>\_and\_<oppia-noninteractive-math math_content-with-value=\"\
+        {&amp;quot;raw_latex&amp;quot;:&amp;quot;1&amp;quot;,&amp;quot;svg_filename&amp;quot;:&amp;quot;&amp;quot;}\"></oppia-noninteractive-math>, respectively, format\
```

Comment:
Okay, I'm not sure how this will look at the end i.e, when we have SVG images for math expressions?

Reply:
We  have to manually add the SVG files in the assets folder and then add the SVG filenames here(similar to images).
We can do that after the SVG generation PR is merged.